lib/policies/load-balancing.js

```diff
@@ -903,6 +923,139 @@ function setLocalDc(lbp, client, hosts) {
     }
 }
 
+/**
+ * This load balancing policy replicates the behavior of the Rust Driver Default Load Balancing Policy.
+ *
+ * It can be configured to be datacenter-aware, rack-aware and token-aware.
+ * Datacenter failover (sending query to a node from a remote datacenter)
+ * for queries with non local consistency mode is also supported.
```

Comment:
AFAIR we changed the logic of DC failover to be possible also for queries with local consistency.
Please update the docs here. If you could be that kind and update Rust docs as well, it would be great :pray: